base.css

```diff
@@ -482,3 +473,16 @@ input[type="checkbox"]::after {
 input[type="checkbox"]:checked::after {
 	left: calc(100% - 1em + 2px);
 }
+
+/* visually hidden, but accessible to assistive tech */
+.visually-hidden {
+  border: 0;
```

Comment:
the indentation looks different here than elsewhere. it looks like you're using spaces while the existing files use tabs

Reply:
Oops, fixed now